packages/angular_devkit/build_angular/src/utils/index-file/inline-critical-css.ts

```diff
@@ -146,7 +146,15 @@ class CrittersExtended extends Critters {
         this.conditionallyInsertCspLoadingScript(document, cspNonce);
       }
 
-      link.prev?.setAttribute('nonce', cspNonce);
```

Comment:
We could've kept this and checked that `link.prev.tagName === 'style'` but that'll be incorrect for a markup like this:

```
<head>
  <style></style>
  <link rel="stylesheet" href="https://some/external/style.css">
</head>
```